.github/workflows/verify.yml

```diff
@@ -37,5 +37,9 @@ jobs:
       - name: Install dependencies
         run: yarn install --frozen-lockfile
 
+        # See https://docs.renovatebot.com/config-validation/
+      - name: Validate renovate config
+        run: npx --yes --package renovate -- renovate-config-validator
```

Comment:
I think we might need to pass in the name of every config that we want to validate as positional arguments. I believe this command will only run against a `renovate.json` by default.

Reply:
Yeah I was afraid that was the case - thanks for the callout - I'll do a loop over all the files